cub/cub/device/device_scan.cuh

```diff
@@ -1165,8 +1165,6 @@ struct DeviceScan
       detail::InputValue<InitValueT>,
       OffsetT,
       AccumT,
-      detail::scan::
-        policy_hub<detail::value_t<InputIteratorT>, detail::value_t<OutputIteratorT>, AccumT, OffsetT, ScanOpT>,
       ForceInclusive::Yes>::Dispatch(d_temp_storage,
```

Comment:
Shouldnt this be moved below?

Reply:
No, because we can take advantage of the default argument of `DispatchScan` now.